src/uct/ib/base/ib_md.c

```diff
@@ -44,6 +44,8 @@ static UCS_CONFIG_DEFINE_ARRAY(pci_bw,
                                sizeof(ucs_config_bw_spec_t),
                                UCS_CONFIG_TYPE_BW_SPEC);
 
+static const char *uct_ib_devx_objs[] = {
+    "rcqp", "rcsrq", "dct", "dcsrq", NULL };
```

Comment:
need to create an enum like UCT_IB_MLX5_DEVX_OBJ_RCQP, UCT_IB_MLX5_DEVX_OBJ_RCSRQ, ...
and initialize this array of strings like this `[UCT_IB_MLX5_DEVX_OBJ_RCQP] = "rcqp",`